packages/common/src/index.ts

```diff
@@ -8,6 +8,7 @@ import * as encoding from './encoding';
 import * as helpers from './type-helpers';
 
 export * from './activity-options';
+export * from './activity-cancellation-details';
```

Comment:
The `ActivityCancellationDetailsHolder` type should not be exposed publicly. It is presently reexported due to this line. Do an `export { specificType1, specificType2 } ...` instead.